app/src/main/java/org/simple/clinic/registration/phone/loggedout/LoggedOutOfDeviceDialog.kt

```diff
@@ -49,23 +42,15 @@ class LoggedOutOfDeviceDialog : AppCompatDialogFragment(), LoggedOutOfDeviceDial
     }
   }
 
-  @Inject
-  lateinit var controller: LoggedOutOfDeviceDialogController
-
   @Inject
   lateinit var effectHandler: LoggedOutOfDeviceEffectHandler
 
   private val okayButton: Button by unsafeLazy {
     (dialog as AlertDialog).getButton(DialogInterface.BUTTON_POSITIVE)
   }
 
-  private val screenDestroys = PublishSubject.create<ScreenDestroyed>()
-  private val onStarts = PublishSubject.create<Any>()
-
   private val events by unsafeLazy {
-    screenCreates()
-        .compose(ReportAnalyticsEvents())
-        .share()
+    Observable.empty<LoggedOutOfDeviceEvent>()
```

Comment:
Please use `Observable.never` here since `empty` is the wrong operator for this use case.